packages/xpath/src/functions/xforms/node-set.ts

```diff
@@ -1,3 +1,5 @@
+import sha256 from 'crypto-js/sha256';
+
 import type { XPathNode } from '../../adapter/interface/XPathNode.ts';
```

Comment:
Minor nits:

```suggestion
import { SHA256 } from 'crypto-js';
import type { XPathNode } from '../../adapter/interface/XPathNode.ts';
```

This change will result in a slightly smaller bundle size, both because it's consistent with the project's other import of the same function and because it potentially improves the bundler's ability to do tree shaking. (In general, named imports are better for that.)

Of course you'd need to update the `sha256` call to uppercase as well. Also note removal of the blank line between imports. Won't matter here, but in general that allows the editor (or whatever in your workflow handles auto-formatting) to auto-sort imports.

- - -

Normally I'd prefer to handle code style issues like the blank line with automated tooling (maybe we can follow up to find an ESLint rule that addresses this?). But it felt worth mentioning while there's a more substantive change to address in the same place.